File: edukate-backend/src/main/java/io/github/sanyavertolet/edukate/backend/dtos/BundleDto.java
package io.github.sanyavertolet.edukate.backend.dtos;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.With;

import java.util.List;

@Data
@AllArgsConstructor
public class BundleDto {
    private String name;
    private String description;
    @With
    private List<String> admins;
    private Boolean isPublic;
    @With
    private List<ProblemMetadata> problems;
    private String shareCode;
}

File: edukate-backend/src/main/kotlin/io/github/sanyavertolet/edukate/backend/dtos/BundleDto.kt
package io.github.sanyavertolet.edukate.backend.dtos

data class BundleDto(
    val name: String,
    val description: String,
    val admins: List<String>,
    val isPublic: Boolean,
    val problems: List<ProblemMetadata>,
    val shareCode: String,
)
